### PR TITLE
updating author view

### PR DESCRIPTION
**Pull request by ghost**
Adding some things to author view:

- making it so when a user types in a non existent user in the search bar, the author view will say 'no user found' instead of just repeating the search text.

- small styling changes to author view

### CODE REVIEW

File: src/helpers/validateId.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const isId = str => /^@[a-zA-Z\/0-9+]+=.ed25519$/g.test(str)
```
Comment (stripedpajamas) [inline src/helpers/validateId.js]: probably use [this regex](https://github.com/ssbc/ssb-ref/blob/master/index.js#L7) to be on the safe side

File: src/components/AuthorView.jsx [hunk 2]
```diff
@@ -74,7 +75,14 @@ class AuthorView extends Component {
               <h1 className='author-view__header' onDoubleClick={this.handleNameClick}>{this.props.author}</h1>
             )
           }
-          <h2 id='author-id'>{this.props.currentAuthorId || this.props.me}</h2>
+          {!isId(this.props.author) && this.props.currentAuthorId === this.props.author
```
Comment (stripedpajamas) [inline src/components/AuthorView.jsx:78]: can you explain this line? i don't understand the `currentAuthorId === author` part

File: src/helpers/validateId.js (hunk 1 above)
Comment (ghost) [reply src/helpers/validateId.js]: done

File: src/components/AuthorView.jsx (hunk 2 above)
Comment (ghost) [reply src/components/AuthorView.jsx:78]: Basically if a user searches for someone that doesn't exist we originally show what they typed both as the user name and as the id. Instead, I decided to check to make sure authorId and author weren't the same thing (meaning a non existent user was searched for and has no actual id) and then display some text to let people know that the user was not found vs. doubling what they searched.

In the case that what they type is a valid id, gester automatically shows the username and id so I didn't need to handle that. But, I am checking if author is an Id in the event they click on someone who hasn't loaded properly and gester is showing their id as their username. I didn't want that to come up as 'user not found'.

Let me know if you need any more clarification.

**Review by stripedpajamas — DISMISSED** [review]

File: src/helpers/validateId.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export const isId = str =>  /^@([A-Za-z0-9\/+]{43}=)\.(?:sha256|ed25519)$/.test(str)
```
Comment (stripedpajamas) [inline src/helpers/validateId.js]: doesn't pass lint anymore lol (two spaces after =>)

**Review by stripedpajamas — APPROVED** [review]

**Pull request merged**